tests/test_stamp.py

```diff
@@ -366,15 +368,15 @@ def new_toplevel_only(self, object_types):
     image, _ = galsim.config.BuildStamp(config, obj_num, logger=logger, do_noise=False)
     print(obj_num, image.center, image.array.shape, image.array.sum())
     assert image.array.shape == (64,64)
-    assert 200_000 < image.array.sum() < 230_000  # 218505
+    assert 230_000 < image.array.sum() < 260_000  # 241_136
 
     # There is also a code path where the xsize,ysize is dictated by the calling routine.
     del config['stamp']['size']
     image, _ = galsim.config.BuildStamp(config, obj_num, logger=logger, do_noise=False,
                                         xsize=128, ysize=128)
     print(obj_num, image.center, image.array.shape, image.array.sum())
     assert image.array.shape == (128,128)
-    assert 290_000 < image.array.sum() < 320_000  # 306675
+    assert 320_000 < image.array.sum() < 350_000  # 338_265
```

Comment:
These all changed because of something in the sims code I suppose?  New version?

Reply:
Not the sims code, this is b/c I added kwargs to imsim-config.yaml to use the measured QE curves in the obs_lsst_data repo instead of the fiducial QE curve from the throughputs repo.  Here's the difference (for R22):
![qe](https://github.com/LSSTDESC/imSim/assets/3650485/7a34b361-016a-4b86-86f2-74aa967b574c)